docs/buildspecs/e4s_testsuite.rst

```diff
@@ -0,0 +1,49 @@
+.. Note:: Please see :ref:`tutorial_setup` before you proceed with this section
+
+.. _buildtest_e4s_testsuite:
+
+Buildtest E4S Testsuite Integration
+===================================
+
+The `E4S Testsuite <https://github.com/E4S-Project/testsuite/>`_ is a collection of lightweight tests assembled for over 100 
+software products provided by `E4S <https://e4s.io>`_ deployments. 
+E4S is a spack based software stack but because not all spack packages provide internal testing functionality it 
+is useful to have an alternative framework for testing deployed software.
+
+The E4S Testsuite operates on a hierarchy of shell scripts with each test containing its own setup, clean, compile 
```

Comment:
either bold or format the following words.

- **setup**, **clean**, **compile**,  and **run** scripts
- `setup`, `clean`, `compile`,  and `run` scripts